web/modules/custom/modal_form_example/modal_form_example.module

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * For adjusting the image width and storing data to database.
+ */
+
+use Drupal\Core\Render\Element;
+
+/**
+ * To adjust the size and height, width for the uploaded image.
+ */
+function modal_form_example_preprocess_image_widget(&$variables) {
```

Comment:
THis will alter all the image widgets, can you please add the relevant conditions here.